### ISSUE TITLE
White theme on android 11 (Possibly others) is buggy

### ISSUE THREAD

**Issue by MV-GH** [dessalines/jerboa#1097]
### Jerboa Version

0.0.41

### Android Version + Phone

Android 11

### Describe The Bug

My android 11 phone is set to dark, maybe this is related. Lots of text stays white, sometimes text does change to black sometimes not. The navigation bar stays blacks. which should be white too.

https://github.com/dessalines/jerboa/assets/67873169/46251a1f-f04a-4f63-87a0-44f9369236ff

### To Reproduce

- Set phone to dark?
- use white theme in Jerboa
- Witness

**Comment by dessalines** [comment]
White theme with dark mode seems like a weird edge-case.

**Comment by MV-GH** [comment]
Not really, lots of people use themes different than the one set as default in the OS. It shouldn't have an effect. on theme of the app.

**Comment by dessalines** [comment]
I can't replicate. Your system theme is dark, lemmy theme mode is light? I'm guessing your device is trying to do that special dark mode for specific apps?

**Comment by MV-GH** [comment]
I think this is android version specific. I don't know hat you mean with special dark mode? But I don't think android 11 does any dynamic theming

**Comment by dessalines** [comment]
Android has this
